operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/util/CommonUtil.java

```diff
@@ -47,10 +48,16 @@ public void checkAllowed(FHIROperationContext operationContext, boolean isImport
             String source = adapter.getStorageProvider();
             String outcome = adapter.getStorageProviderOutcomes();
 
-            boolean s = ConfigurationFactory.getInstance().hasStorageProvider(source);
-            boolean o = ConfigurationFactory.getInstance().hasStorageProvider(outcome);
-            if (!s || !o) {
-                throw FHIROperationUtil.buildExceptionWithIssue("The storage provider for outcome [" + s + "] or source [" + o + "] passed is not configured properly" , IssueType.EXCEPTION);
+            if (!ConfigurationFactory.getInstance().hasStorageProvider(source)
+                    || !ConfigurationFactory.getInstance().hasStorageProvider(outcome)) {
+                String fhirTenant = FHIRRequestContext.get().getTenantId();
+                StringBuilder builder = new StringBuilder("The 'fhirServer/bulkdata/storageProvider' for source [");
```

Comment:
still not a fan of the term "source" for this.
in the config we settled on:
* defaultExportProvider
* defaultImportProvider
* defaultOutcomeProvider

so maybe use "exportProvider" here (or reword so that we can just say "storageProvider 'xyz' is missing configuration"?

Reply:
it's used in both import and export

Reply:
we'd have to change the signature